bats/helper/empty-strings-null-values-sch1.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Are we still doing schema.json files? I think these will be ripped out soon. You should specify test table schemas with SQL create table commands

Reply:
I updated the current test "_create a table with null values from csv import with json file_" in **create-tables.bats** so that the current JSON schema file had `-sch` appended to it. The test makes use of the `import -c` argument, hence the need for the JSON file. To minimize differences between the tests, I just re-used the JSON schema that was already there, and made a new one by making slight modifications.

Reply:
I would just use a sql -q command to create the table, followed by import -u.